include/utility/json_utility.hpp

```diff
@@ -206,15 +206,6 @@ inline bool executePostFailAction(const nlohmann::json& i_parsedConfigJson,
         return false;
     }
 
-    if (!(i_parsedConfigJson["frus"][i_vpdFilePath].at(0).contains(
```

Comment:
Please add this check back. This is an utility API. It's not gauranteed that every other callers of this API will have this if check from their end.

Reply:
No, Not required now as it will result in double execution. 
Also, Post fail action is only required as a part of pre-action failure, it should not be blindly executed without that flow as it will not be known in what context this API is being called.

In case this utility is called independently by another API, then the caller should have intelligence to know what needs to be done in case of failure from this API.

Reply:
in line 210 [ ] operator will throw exception if PostFailAction is missing in JSON.

Reply:
Agree, why this check should be removed, Ideally caller should call API after checking for action required but no issue in having a double check in case the caller fails to do the check before call and calls directly.

Reply:
so we need the if check if (!(i_parsedConfigJson["frus"][i_vpdFilePath].at(0).contains( "PostFailAction")))

Reply:
there are many ways to do this , why would I call this function if the function is designed to use the specific tag?  the question arise that if the tag is missing is it a fault or not , it becomes difficult to attain for who uses it and how , so instead of fixing something by a check inside , lettuce check remain outside and maybe just return true if the tag is not there.

I thought I told you to remove the log and make the return true.
you should also make these defines on the functions oxygen also .

Reply:
create a ISSUE for this as we will debate on this flow later